app/helpers/miq_ae_class_helper.rb

```diff
@@ -324,4 +324,26 @@ def datastore_form(ae_ns, sb_data, type)
                             :nameReadOnly  => domain && !ae_ns.editable_property?(:name),
                             :descReadOnly  => domain && !ae_ns.editable_property?(:description)})
   end
+
+  def first_user_summary(ae_method, sb)
+    rows = [
+      row_data(_('Type'), ae_method.location),
+      row_data(_('Fully Qualified Name'), sb[:namespace_path]),
+      row_data(_('Name'), ae_method.name),
+      row_data(_('Display Name'), ae_method.display_name),
+      row_data(_('Created On'), format_timezone(@ae_method.created_on, Time.zone, "gtl")),
```

Comment:
please remove `@` from `@ae_method `since it is being used in method param

Reply:
done